docs/topics/multiplatform-mobile/multiplatform-mobile-create-first-app.md

```diff
@@ -1,66 +1,75 @@
-    >
-    {type="note"}
+## Create the project from template
 
-You can also watch a video of this tutorial created by Ekaterina Petrova, Kotlin Developer Advocate.
-
-<video href="GcqFhoUuNNI" title="Kotlin Multiplatform Multiverse, Episode 2: Your First Kotlin Multiplatform Mobile App Tutorial"/>
+> You can also watch a [video of this tutorial](https://www.youtube.com/watch?v=GcqFhoUuNNI) created by Ekaterina Petrova, Kotlin Product Marketing Manager.
+>
+{type="tip"}
 
-2. In Android Studio, select **File** | **New** | **New Project**.
-3. Select **Kotlin Multiplatform App** in the list of project templates, and click **Next**.  
+1. In Android Studio, select **File | New | New Project**.
+2. Select **Kotlin Multiplatform App** in the list of project templates, and click **Next**.  
 
     ![Mobile Multiplatform project template](multiplatform-mobile-project-wizard-1.png)
     
-4. Specify a name for your first application, and click **Next**.  
+3. Specify a name for your first application, and click **Next**.  
 
     ![Mobile Multiplatform project - general settings](multiplatform-mobile-project-wizard-2.png)
 
-5. In the window that opens, do the following:
-   * Keep the default names for the application and shared folders.
-   * Select the checkbox to generate sample tests for your project. 
-   * Select **Regular framework** in the list of iOS framework distribution options. 
-     
-   Click **Finish** to create a new project.
+4. In the **iOS framework distribution** list, select **Regular framework**.
 
-    ![Mobile Multiplatform project - additional settings](multiplatform-mobile-project-wizard-3.png)
-    
-    > If you want to use Kotlin Multiplatform module as a CocoaPods dependency, select the **CocoaPods dependency manager** option.
-    > To learn more about CocoaPods dependencies, see [CocoaPods integration](native-cocoapods.md).
-    >
-    {type="note"}
-    
-Now wait while your project is set up. It may take some time to download and set up the required components when you 
+   ![Mobile Multiplatform project - additional settings](multiplatform-mobile-project-wizard-3.png)
+
+5. Keep the default names for the application and shared folders. Click **Finish**.
+
+Wait for the project to set up. It may take some time to download and set up the required components when you 
 do this for the first time.
-    
-To view the complete structure of your mobile multiplatform project, switch the view from **Android** to **Project**. 
-You can [understand the project structure](multiplatform-mobile-understand-project-structure.md) and how you can use this. 
-    
+
+## Examine the project structure
+
+To view the complete structure of your mobile multiplatform project, switch the view from **Android** to **Project**.
+
 ![Select the Project view](select-project-view.png){width=200}  
+
+There are three modules in a Kotlin Mobile Multiplatform project:
```

Comment:
I’d use the autogenerated module names, otherwise it reads a bit weird (I mean, the repetition of ‘Android application’ is a bit confusing here: <<Android application is a Kotlin module that builds into an Android application>>; the first one is a title, but it’s unclear whether we define it recursively or refer to a module name which is different).

Also, I’d add comments on how android and iOS apps use the shared module.

Something like:

_shared_ module is a Kotlin module that contains common logic for both Android and iOS applications. It builds into both an Android library and an iOS framework. It uses [Gradle](https://branch-kmm-onboarding.kotlin-web-site.labs.jb.gg/docs/gradle.html) as a build system that helps you automate your building process.

_androidApp_ is a Kotlin module that builds into an Android application. It uses Gradle as a build system. It depends on and uses the shared module as a regular Android library.

_iosApp_ is an Xcode project that builds into an iOS application. It depends on and uses the shared module as an iOS framework. It might either use it as a regular framework or via CocoaPods dependency based on what we’ve chosen in the previous step for “iOS framework distribution”. In this tutorial, it’s a regular framework dependency.